int32 ts
float32 x	# m
float32 y	# m
float32 height	# m
float32 velx	# m/s
float32 vely	# m/s
float32 velz	# m/s
